Dockerfile

```diff
@@ -749,6 +752,42 @@ RUN \
   cd zimg-* && ./autogen.sh && ./configure --disable-shared --enable-static && \
   make -j$(nproc) install
 
+#bump: libjxl /LIBJXL_VERSION=v([\d.]+)/ https://github.com/libjxl/libjxl.git
```

Comment:
Pipeline can be something like `https://github.com/libjxl/libjxl.git|^0` i think